packages/expo-analytics-segment/ios/EXSegment/EXSegment.m

```diff
@@ -1,19 +1,26 @@
 // Copyright 2015-present 650 Industries. All rights reserved.
 
 #import <EXSegment/EXSegment.h>
-
+#import <EXConstantsInterface/EXConstantsInterface.h>
 #import <SEGAnalytics.h>
 
+static NSString * const EXSegmentOptOutKey = @"EXSegmentOptOutKey";
+
 @interface EXSegment ()
 
 @property (nonatomic, strong) SEGAnalytics *instance;
+@property (nonatomic, weak) id<EXConstantsInterface> constants;
 
 @end
 
 @implementation EXSegment
 
 EX_EXPORT_MODULE(ExponentSegment)
 
+- (void)setModuleRegistry:(EXModuleRegistry *)moduleRegistry
+{
+  _constants = [moduleRegistry getModuleImplementingProtocol:@protocol(EXConstantsInterface)];
```

Comment:
I believe we should introduce some `assertions` for retrieving modules from registry. What you say?